whylog_vim/output_formater/output_aggregator.py

```diff
@@ -16,21 +21,26 @@ def get_content(self):
         # The same type as vim buffor.
         return self.output_lines
 
-    def create_button(self, callback_function, line=None):
+    def create_button(self, callback_function, function_meta, line=None):
         line = line or self._get_current_line_number()
-        self.buttons[line] = callback_function
+        self.function_lines[function_meta] = line
+        self.callbacks[line] = callback_function
 
     def set_default_callback_function(self, default_callback_function):
         self.default_callback_function = default_callback_function
 
+    def call_by_menu_function(self, menu_function):
```

Comment:
In the future we will add shortcuts for every operation of teacher's menu. If someone will wants to edit line content, he will press `el0c` (edit line 0 content) and the whylog proxy will call function this function with parametr (FunctionNames.EDIT_LINE_CONTENT, 0).